src/styles/main.scss

```diff
@@ -1,3 +1,15 @@
+* {
+  box-sizing: border-box;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  color: black;
```

Comment:
It's better to use a variable for the color to make the code more maintainable. You can declare it in 'variables.scss'.